plugins/modules/system/python_requirements_info.py

```diff
@@ -121,7 +121,8 @@ def main():
             python_version=sys.version,
             python_system_path=sys.path,
         )
-    pkg_dep_re = re.compile(r'(^[a-zA-Z][a-zA-Z0-9_-]+)(==|[><]=?)?([0-9.]+)?$')
+    pkg_dep_re = re.compile(
+        r'(^[a-zA-Z][a-zA-Z0-9_-]+)(\[[a-zA-Z]+\])?(==|[><]=?)?([0-9.]+)?$')
```

Comment:
It might be interesting to allow white spaces, leading and trailing, as well as around the operator.